src/test/java/fi/metatavu/metaform/test/functional/tests/VersionTestsIT.kt

```diff
@@ -0,0 +1,131 @@
+package fi.metatavu.metaform.test.functional.tests
```

Comment:
There should be permission tests, that assert that only users with proper permissions may access these endpoints